draft-ietf-scitt-architecture.md

```diff
@@ -752,48 +781,6 @@ RegistrationInfo = {
 }
 ~~~
 
-## Signed Statement Issuance
```

Comment:
This section was duped, and out of sync with [line 689](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/118/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1R689)